docs/topics/api.rst

```diff
@@ -154,6 +155,17 @@ Settings API
 .. autoclass:: BaseSettings
    :members:
 
+.. _topics-api-addonmanager:
+
+AddonManager API
+================
```

Comment:
I think we can remove this section, as AddonManager is an internal component. This section contains classes which users can either use directly, or override via settings with custom implementations; it doesn't seem that AddonManager fits.